feature-group-definitions/grid.yml

```diff
@@ -0,0 +1,30 @@
+spec: https://drafts.csswg.org/css-grid-3/
+caniuse: css-grid
+compat_features:
```

Comment:
I think we're missing the `place-content`, `place-items`, `place-self`, `align-content`, and `justify-content` properties.
You've got `self` and `items` for both `justify` and `align` directions, but you also need `content` (which applies to a grid container element and applies to the alignment "subjects" in the grid, aka the row and column tracks). And you need `place` too, which is the short-hand for `align` and `justify`.

Reply:
If the data is structured correctly the question becomes: should we treat aliases or prefixes as full support, and making the feature eligible for baseline, or should we treat it like an important note that should be called out when support is summarized? I'd lean towards the latter.

Reply:
That's a good question! I've opened https://github.com/web-platform-dx/feature-set/issues/86 to preserve it (and to raise some closely related questions).